dash_labs/plugins/pages.py

```diff
@@ -278,12 +277,16 @@ def _infer_path(filename, template):
         return "/".join(default_template_path)
 
 
-def _import_layouts_from_pages():
+def _import_layouts_from_pages(app):
     for (root, dirs, files) in os.walk("pages"):
         for file in files:
+            page_filename = os.path.join(root, file).replace("\\", "/")
+            if file.endswith(".md"):
+                _register_page_from_markdown_file(page_filename, app)
```

Comment:
For consistency with the Python file version, I think we should only register a page if it contains the front-matter declaring registration.

Otherwise, the markdown files are implicitly imported whereas the python files have to have `register_page` which is inconsistent.

This allow allows users to include markdown snippets in other python pages, or other markdown pages, without having those snippets be registered as pages.

Reply:
Ok, this is done now too